deploy/README.md

```diff
@@ -60,4 +60,5 @@ The following matrix displays the tested package versions for our Helm chart.
 Sumo Logic Helm Chart | Prometheus Operator | Fluent Bit | Falco
 |:-------- |:-------- |:-------- |:--------
 0.6.0 - 0.8.0 | 6.2.1 | 2.4.4 | 1.0.5
-0.9.0 - Latest | 6.2.1 | 2.4.4 | 1.0.8
+0.9.0 - 0.11.0 | 6.2.1 | 2.4.4 | 1.0.8
+0.12.0 - Latest | 8.2.0 | 2.8.1 | 1.0.9
```

Comment:
Can we reorder these rows? so the latest row shows up on top, and oldest rows are on bottom